core/src/main/scala/com/phaller/rasync/HandlerPool.scala

```diff
@@ -28,6 +27,9 @@ class HandlerPool(parallelism: Int = 8, unhandledExceptionHandler: Throwable =>
 
   private val cellsNotDone = new AtomicReference[Map[Cell[_, _], Queue[SequentialCallbackRunnable[_, _]]]](Map()) // use `values` to store all pending sequential triggers
 
+  private var interruptLatch = new CountDownLatch(1)
```

Comment:
Should also be `@volatile` due to assignment on line 466 below.

Reply:
Done